backend/src/users/users.service.spec.ts

```diff
@@ -60,10 +71,12 @@ describe('UsersService', () => {
 
   describe('when finding all Users', () => {
     it('should return array of User', async () => {
-      const messages = await service.findAll();
-      expect(messages).toBeInstanceOf(PageDto);
-      if (messages.data.length !== 0)
-        expect(messages.data[0]).toBeInstanceOf(MockUserEntity);
+      jest
+        .spyOn(repository, 'findAndCount')
+        .mockReturnValue(Promise.resolve([[new User()], 1]));
+      const users = await service.findAll();
+      expect(users).toBeInstanceOf(PageDto);
+      if (users.data.length !== 0) expect(users.data[0]).toBeInstanceOf(User);
```

Comment:
```suggestion
      const messages = await service.findAll();
      expect(messages).toBeInstanceOf(PageDto);
      if (messages.data.length !== 0)
        expect(messages.data[0]).toBeInstanceOf(MockUserEntity);
```
Works with the suggested change of `MockRepository`.